src/service/types.ts

```diff
@@ -176,13 +178,15 @@ export type VehiclesQuery = {
   lon: number;
   range: number;
   formFactors?: FormFactor | FormFactor[];
+  operators?: string[];
```

Comment:
Joi schema validated the `operator` field for mobility vehicles and stations, but it wasn't included in the types.